src/amo/components/Icon/styles.scss

```diff
@@ -290,33 +249,10 @@
   @include permission-icon($name: 'navigation');
 }
 
-.Icon-developer {
-  @include icon($name: 'developer');
-}
-
-.Icon-artist {
-  @include icon($name: 'artist');
-}
```

Comment:
We need this one in the user profile of a (theme) artist, e.g., https://addons-dev.allizom.org/en-US/firefox/user/8699116/